hadoop-ozone/ozonefs-shaded/pom.xml

```diff
@@ -187,6 +304,87 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>${maven-dependency-plugin.version}</version>
+        <executions>
+          <execution>
+            <id>unpack-dependencies</id>
+            <goals>
+              <goal>unpack-dependencies</goal>
+            </goals>
+            <phase>validate</phase>
+            <configuration>
+              <includeGroupIds>io.netty</includeGroupIds>
+              <includes>**/META-INF/native/*</includes>
+              <includeArtifactIds>netty-resolver-dns-native-macos,
+                netty-tcnative-boringssl-static,
+                netty-transport-native-epoll,
+                netty-transport-native-kqueue</includeArtifactIds>
+              <outputDirectory>${project.build.directory}/classes/</outputDirectory>
+              <overWriteReleases>true</overWriteReleases>
+              <overWriteSnapshots>true</overWriteSnapshots>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <!-- The native libraries need to be named in the same way we relocate the Netty classes via the shade-plugin -->
+      <!-- We shaded ratis thirdparty, so the dynamic library name here includes both ratis thirdparty shaded and ozone shaded-->
+      <plugin>
+        <groupId>com.coderplus.maven.plugins</groupId>
+        <artifactId>copy-rename-maven-plugin</artifactId>
+        <version>${copy-rename-maven-plugin.version}</version>
```

Comment:
nit: please define version in `pluginManagement` in root `pom.xml`, similar to other plugins:

https://github.com/apache/ozone/blob/8489cc86b5bc23eb17e1e85b4a2d3103d7e2a1ce/pom.xml#L1331-L1337

Reply:
Done.